src/compiler/scala/tools/nsc/transform/Erasure.scala

```diff
@@ -495,11 +495,13 @@ abstract class Erasure extends InfoTransform
     }
 
     /** Check that a bridge only overrides members that are also overridden by the original member.
-     *  This test is necessary only for members that have a value class in their type.
+     *  This test is necessary only* for members that have a value class in their type.
      *  Such members are special because their types after erasure and after post-erasure differ/.
      *  This means we generate them after erasure, but the post-erasure transform might introduce
      *  a name clash. The present method guards against these name clashes.
      *
+     *  *Note: Also check bridges with `Object`-typed params, as users might write that signature in a parent.
+     *
```

Comment:
Instead of using a footnote, I think it would be better to just revise the comment to reflect the current status.

Reply:
Technically, a footnote would go in the page footer. I think they have `@implNote` or similar? in Java is it. I see a stray asterisk, a planetary punctuation.